docs/content/docs/how-to/writing-tables.md

```diff
@@ -314,7 +314,10 @@ Run the following command to submit a 'merge-into' job for the table.
     --warehouse <warehouse-path> \
     --database <database-name> \
     --table <target-table> \
+    [--target-as <target-table-alias>] \
     --using-table <source-table> \
```

Comment:
`--source-table`

Reply:
Keep this because the sql syntax is `MERGE INTO T USING S`